jsondrop.js

```diff
@@ -6,15 +6,23 @@ const defaultConfig = {
 
 /**
  * Renders a JavaScript object into a pretty HTML JSON-esque representation.
+ * @param {*} value - The value to be rendered. Non-standard objects will only render their object
+ *					  type as a string.
+ * @param {Object} config - The formatter configuration to use. Properties:
```

Comment:
`config` here should be marked as an optional parameter
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values